UnityMcpBridge/UnityMcpServer~/src/tools/manage_script_edits.py

```
@@ -0,0 +1,815 @@
+            if not text.endswith("\n"):
+                text += "\n"
+        elif op == "anchor_insert":
+            anchor = edit.get("anchor", "")
+            position = (edit.get("position") or "before").lower()
+            insert_text = edit.get("text", "")
+            flags = re.MULTILINE | (re.IGNORECASE if edit.get("ignore_case") else 0)
+            m = re.search(anchor, text, flags)
+            if not m:
+                if edit.get("allow_noop", True):
+                    continue
+                raise RuntimeError(f"anchor not found: {anchor}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)